examples/ball.js

```diff
@@ -19,8 +20,18 @@ import {Become, Animate, Translate, RepeatSequence} from '../node/animations.js'
 
 const SOURCE = "https://github.com/Monadical-SAS/redux-time/blob/master/examples/ball.js"
 
+const initial_ball_state = {following: false}
+const ballReducer = (state=initial_ball_state, action) => {
+    console.log({state, action})
+    switch (action.type) {
+        case 'BALL_SET_FOLLOWING':
```

Comment:
If we're trying to set a good example (also eslint will error without the curly's):

```javascript
        case 'BALL_SET_FOLLOWING': {
            return {following: action.following}
        }
        default: {
            return state
        }
```